src/js/game/LevelMVC/LevelModel.js

```diff
@@ -597,9 +597,29 @@ module.exports = class LevelModel {
         return false;
       }
     }
+
+    if (this.checkConflict(this.getMoveDirectionPosition(entity, direction), plane)) {
+      return false;
```

Comment:
Would be great to have unit test coverage of this line!

Reply:
Roger that 👍